overviews/quasiquotes/expression-details.md

```diff
@@ -75,20 +75,20 @@ Each term identifier is defined by its name and by the fact of being backquoted
     scala> val backquoted = q"`$name`"
     backquoted: universe.Ident = `Foo`
 
-Although backquoted and non-backquoted identifiers may refer to the same things they are not equivalent from syntactical point of view:
+Although backquoted and non-backquoted identifiers may refer to the same thing they are not equivalent from a syntactical point of view:
```

Comment:
Although backquoted and non-backquoted identifiers may refer to the same thing, they are not syntactically equivalent: